engine/testutil/nodes.go

```diff
@@ -610,7 +611,7 @@ func ExecutionNode(t *testing.T, hub *stub.Hub, identity bootstrap.NodeInfo, ide
 	require.NoError(t, err)
 
 	registerDir := unittest.TempPebblePath(t)
-	pebbledb, err := storagepebble.OpenRegisterPebbleDB(registerDir)
+	pebbledb, err := storagepebble.OpenRegisterPebbleDB(sutil.NewLogger(node.Log, "pebbledb-registers"), registerDir)
```

Comment:
Why is the caller always responsible for creating a logger compatible with pebble? Why is that not done inside of the pebble/badger module. The caller could simply pass in a zerolog.Logger (and optionally do it with a component `node.Log.With().Str("component", comp).Logger()`.

Reply:
Good point 👍 

updated 96c625c02a